rudder-server/SOURCES/rudder-jetty.default

```diff
@@ -23,19 +23,26 @@ JAVA_MAXPERMSIZE=256
 #
 #JAVA_OPTIONS=""
 
-# Java Garbage collector option
-# On large heap (6 Go or more) and loaded platforms, ConcurrentMarkSweep (default GC 
-# for Rudder) becomes inefficient and Rudder can experience long "stop the world" 
-# garbage collections.
-# To avoid that, you can use G1GC which leads to faster (or no) full GC on big heap,
-# at the prize of a less throughout and a need of some more memory to work 
-# efficiently (~10%).
-# You can also tune you GC option here.
+# Java garbage collector option
+#
+# By default, Rudder uses G1GC which is a very good general purpose GC on
+# all sizes of heap. The configuration should be good on most case.
+# But you can change GC or tune GC options here, like debug flags.
+# Documentation can be found here:
+# https://docs.oracle.com/en/java/javase/11/gctuning/garbage-first-garbage-collector-tuning.html
 #
 #JAVA_GC="-XX:+UseG1GC
-#-XX:+UnlockExperimentalVMOptions
-#-XX:+UseStringDeduplication
```

Comment:
we should probably keep our default conf as basis here

Reply:
I don' understand ? Do you want that we keep the `rudder-jetty.default` as it was in previous versions ?

Reply:
default of rudder-jetty.conf should be reflected in this file, so that we know what's the default config; and offer as an extra others configuration